pkg/image/api/helper.go

```diff
@@ -150,8 +166,8 @@ func (r DockerImageReference) DaemonMinimal() DockerImageReference {
 		r.Namespace = ""
```

Comment:
Hmm this seems to be another bug. It should be cleared only in case of Docker registry. If I only noticed it earlier :-(. I'll create a follow-up.